android/app/src/main/java/com/zulipmobile/MainActivity.kt

```diff
@@ -28,6 +29,45 @@ open class MainActivity : ReactActivity() {
         }
     }
 
+    /* Returns true just if we did handle the intent. */
+    private fun maybeHandleIntent(
+        intent: Intent?,
+        contentResolver: ContentResolver
```

Comment:
This parameter can be simplified out in much the same way as `application: ReactApplication`: the call sites just get it from a getter on `this`, so this function can do the same thing.